config/vm.args

```diff
@@ -1,4 +1,4 @@
--sname aeternity@localhost
+-sname epoch@localhost
```

Comment:
Tested re DB name in integration environment
Handling of node name in DB [tracked](https://www.pivotaltracker.com/story/show/163423623).